<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>glaxnimate::gui::KeyframeEditorWidget</class>
 <widget class="QWidget" name="KeyframeEditorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>447</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="glaxnimate::gui::KeyframeTransitionWidget" name="bezier_editor" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Presets</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_before">
        <item>
         <property name="text">
          <string>Hold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ease In</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Finish</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_after">
        <item>
         <property name="text">
          <string>Hold</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Linear</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ease Out</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Custom</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>glaxnimate::gui::KeyframeTransitionWidget</class>
   <extends>QWidget</extends>
   <header>widgets/timeline/keyframe_transition_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>combo_before</sender>
   <signal>activated(int)</signal>
   <receiver>KeyframeEditorWidget</receiver>
   <slot>preset_before(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>403</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>483</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>combo_after</sender>
   <signal>activated(int)</signal>
   <receiver>KeyframeEditorWidget</receiver>
   <slot>preset_after(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>preset_before(int)</slot>
  <slot>preset_after(int)</slot>
 </slots>
</ui>
